client/src/app/global-components/accounts/accounts.component.html

```diff
@@ -0,0 +1,120 @@
+<div class="container">
+  <ul class="accounts-list">
+    <p
+      [ngClass]="{active: !accounts.length}"
```

Comment:
If you have only one conditional class use `[class.active]="!accounts.length"`

Reply:
I changed